src/lib.rs

```diff
@@ -287,8 +265,14 @@ impl<C> ProxyConnector<C> {
     /// Create a new secured Proxies
     #[cfg(feature = "rustls")]
     pub fn new(connector: C) -> Result<Self, io::Error> {
-        let cfg = Arc::new(tokio_rustls::rustls::ClientConfig::new());
+        let mut config = tokio_rustls::rustls::ClientConfig::new();
+
+        config.root_store = rustls_native_certs::load_native_certs()
+            .expect("cannot access native cert store");
```

Comment:
Why not returning an `Error` here?

Reply:
Mainly because `hyper-tls` has panic. I think it may fail only if some root certificates are not installed in the system, and it has no sense to handle this error (except the case when system built with other native-tls, but why do need rustls in that case)

Reply:
I can imagine scenarios where you want to try supporting tls and revert to non-tls if it doesn't work ... All without panicking

Reply:
Yep, fixed